beacon-chain/forkchoice/doubly-linked-tree/forkchoice.go

```diff
@@ -397,6 +447,10 @@ func (f *ForkChoice) UpdateJustifiedCheckpoint(jc *forkchoicetypes.Checkpoint) e
 	f.store.checkpointsLock.Lock()
 	defer f.store.checkpointsLock.Unlock()
 	f.store.justifiedCheckpoint = jc
+	bj := f.store.bestJustifiedCheckpoint
```

Comment:
Do we still need `UpdateJustifiedCheckpoint` when everything is done?

Reply:
It's not entirely clear yet. In principle it can be gone, and I am hoping that it completely disappears, but I haven't yet cleaned up init sync, where we don't really have the state cause we do transition in batches and